src/SDKs/DataFactoryV1Extended/Microsoft.Azure.Management.DataFactories.Extended/Microsoft.Azure.Management.DataFactories.Extended.csproj

```diff
@@ -0,0 +1,13 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.DataFactories.Extended</PackageId>
```

Comment:
Previous version is called Microsoft.Azure.Management.DataFactories, which is our ADF V1 product. Here are two reasons to add this new extended one: 
1. This change is based on V1 product which introduces many enhancement for V1 APIs, but we don't want to break previous APIs, so it's a new extended API set which can be used if customer need to use those enhanced feature. 
2. Azure SDK don't accept Hydra code any more, so we have to use swagger to generate code for new features. But our V1 APIs are hydra based and it's risky and resource consuming work to migrate them to swagger, and we are more and more focus on our V2 product.

So we decide to have this extended proj to avoid breaking existing APIs and also have the extension point to add new APIs for v1 product later if there is.